google-cloud-spanner/src/main/java/com/google/cloud/spanner/TransactionManagerImpl.java

```diff
@@ -53,8 +53,20 @@ public void setSpan(ISpan span) {
   @Override
   public TransactionContext begin() {
     Preconditions.checkState(txn == null, "begin can only be called once");
+    return begin(ByteString.EMPTY);
+  }
+
+  @Override
+  public TransactionContext begin(AbortedException exception) {
+    Preconditions.checkState(txn == null, "begin can only be called once");
+    ByteString previousAbortedTransactionID =
```

Comment:
Here also: Add a not-null check for the AbortedException.

Reply:
Done